pallets/gear/src/benchmarking/mod.rs

```diff
@@ -33,6 +33,7 @@
 //! So, we suppose this instruction have weight 0.
 
 #![cfg(feature = "runtime-benchmarks")]
+#![allow(clippy::assertions_on_constants)]
```

Comment:
this is bug in current clippy version (see #3452), plz delete it

Reply:
> plz delete it

It's just impossible to use something else but bash scripts which is inconvenient for my workflow

Reply:
https://doc.rust-lang.org/cargo/reference/workspaces.html#the-lints-table

Reply:
Not supported by our pinned toolchain. Lints were merged 9 September. Our toolchain is 5 September

https://github.com/rust-lang/cargo/pull/12648#event-10323314352